### PR TITLE
Update to rails 6.1

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Latest is greatest

## How was this change tested?
Unit

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,8 +3,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6.0.2', '>= 6.0.2.1'
+gem 'rails', '~> 6.1'
```
Comment (ndushay) [inline Gemfile:6]: in discussion earlier, we said pin to patch level as rails isn't great at avoiding glitches in backwards compatibility

File: bin/rails [hunk 2]
```diff
@@ -1,11 +1,7 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-begin
-  load File.expand_path('spring', __dir__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
-end
```
Comment (ndushay) [inline bin/rails:8]: dumb question:  I don't think this exception handling was part of rails boilerplate before so ... do we want to keep it?

**Review by ndushay — CHANGES_REQUESTED** [review]
One request for a change, one question.

File: bin/rails (hunk 2 above)
Comment (justinlittman) [reply bin/rails:8]: No idea. However, I can't think of any reason that this app would need special handling here.

File: Gemfile (hunk 1 above)
Comment (justinlittman) [reply Gemfile:6]: Ahhh.

File: bin/rails (hunk 2 above)
Comment (jcoyne) [reply bin/rails:8]: This was just what rails generated when we did `rails new`. I don't think there's any reason not to accept this.

File: Gemfile (hunk 1 above)
Comment (mjgiarlo) [reply Gemfile]: Yes, Rails does not follow semver.

**Review by ndushay — APPROVED** [review]

**Comment by ndushay** [comment]
I approved this PR;  @justinlittman merge and deploy at will.

**Pull request merged**